eva_cttv_pipeline/config.py

```diff
@@ -11,7 +11,7 @@
 #########################################
 # cellbase
 BATCH_SIZE = 200
-HOST = 'www.ebi.ac.uk'
+HOST = 'wwwdev.ebi.ac.uk'
```

Comment:
Yes, the Clinvar collection in the production Cellbase is the same data that was used in the February 2016 batch. The data used in this batch (i.e. the newly loaded Clinvar data) is in development, according to Pablo.